src/cmd/build.js

```diff
@@ -60,8 +61,21 @@ function defaultPackageCreator(
           filter: (...args) => fileFilter.wantFile(...args),
         })
         .then((buffer) => {
+          let messageData: any;
+          if (manifestData.default_locale) {
+            messageData = readFileSync(
+               path.join(sourceDir, '_locales', manifestData.default_locale,
+                'messages.json'));
+          }
+          let extensionName: string;
+          if (messageData) {
+            extensionName = JSON.parse(messageData).extensionName.description;
```

Comment:
I'd happened to start playing around with this before seeing this PR. For the matching I was using `/__MSG_([A-Za-z0-9@_]+?)__/` as that's what Firefox actually uses.

You'll also want to use the second argument to the function that's passed to `replace` rather than `matches[1]`.